.github/actions/schema-validator/index.js

```diff
@@ -0,0 +1,41 @@
+const core = require("@actions/core");
+const dirtyJSON = require("dirty-json"),
+    {Validator} = require("jsonschema"),
+    schemaType = process.env.ID,
```

Comment:
If, for any reason, the env var `ID` is undefined, the test will fail. In that case, should we assign a default value to it?